source/replication/site-replication-overview.rst

```diff
@@ -0,0 +1,532 @@
+
+=========================
+Site Replication Overview
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Site replication configures multiple independent MinIO deployments as a cluster of replicas called peer sites.
+
+Site replication assumes the use of either the included MinIO identity provider (IDP) *or* an external IDP.
+All configured deployments must use the same IDP.
+Deployments using an external IDP must use the same configuration across sites.
+
+Overview
+--------
+
+.. _minio-site-replication-what-replicates:
+
+What Replicates Across All Sites
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/common-replication.rst
+   :start-after: start-mc-admin-replicate-what-replicates
+   :end-before: end-mc-admin-replicate-what-replicates
+
+What Does Not Replicate Across Sites
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/common-replication.rst
+   :start-after: start-mc-admin-replicate-what-does-not-replicate
+   :end-before: end-mc-admin-replicate-what-does-not-replicate
+
+
+Initial Site Replication Process
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After enabling site replication, identity and access management (IAM) settings sync in the following order:
+
+.. tab-set::
+
+   .. tab-item:: MinIO IDP
+
+      #. Policies
+      #. User accounts (for local users)
+      #. Groups
+      #. Service accounts
+         
+         Service accounts for ``root`` do not sync.
+
+      #. Policy mapping for synced user accounts
+      #. Policy mapping for `Security Token Service (STS) users <https://docs.min.io/docs/minio-sts-quickstart-guide.html>`
+
+   .. tab-item:: OIDC
+
+      #. Policies
+      #. Service accounts
+         
+         Only service accounts related to user accounts with valid policies for the MinIO deployment.
+         Service accounts for ``root`` do not sync.
+
+      #. Policy mapping for synced user accounts
+      #. Policy mapping for `Security Token Service (STS) users <https://docs.min.io/docs/minio-sts-quickstart-guide.html>`
+
+   .. tab-item:: LDAP
+
+      #. Policies
+      #. Groups
+      #. Service accounts
+
+         Only service accounts related to user accounts with valid policies for the MinIO deployment.
+         Service accounts for ``root`` do not sync.
+
+      #. Policy mapping for synced user accounts
+      #. Policy mapping for `Security Token Service (STS) users <https://docs.min.io/docs/minio-sts-quickstart-guide.html>`
+
+
+Site Healing
+~~~~~~~~~~~~
+
+With site replication enabled, for any item :ref:`supported in site replication <minio-site-replication-what-replicates>`, the MinIO deployment resynchronizes that item from the peer site with the most updated ("latest") version of that item.
+
+
+Prerequisites
+-------------
+
+One Site with Data at Setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Only *one* site can have data at the time of setup.
+The other sites must be empty of buckets and objects.
```

Comment:
Would it require being empty of all replica-eligible data, or *just* bucket/objects? e.g. if two sites have IAM data persisted, what happens.

Reply:
Confirmed that it is just buckets and objects.
IAM data can exist and differ between the sites prior to enabling site replication, and replication occurs just fine (including sharing all IAM data between sites, for example).